cst438week1_assignment1/src/main/java/cst438/Movie.java

```diff
@@ -27,18 +25,16 @@
 		
 		@NotNull
 		@Max(100)
+		@Min(0)
```

Comment:
```suggestion
		@Min(1)
```
Probably change this to have the range for rating between 1 and 5